ES/ESdd009.xml

```diff
@@ -165,132 +266,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              of the incipits of daily readings and their sections in <ref target="#ms_i1 #ms_i2 #ms_i3">Texts I-III</ref>; text
+                           dividers; elements of the punctuation signs; elements of Ethiopic numerals.</seg>                                             
                      </handNote>
-                     <handNote xml:id="h2" script="Ethiopic">The quality of the handwriting varies
-                        slightly throughout the Ms. (from <locus target="#56r">56r</locus> it
-                        becomes more
-                        right-sloping).<!--The names of this hands need to be replaced by appropriate refs.--></handNote>
                   </handDesc>
+                  
                   <decoDesc>
-
                      <decoNote type="frame" xml:id="d1">
-                        <locus target="#113r">113r</locus>: Simple coloured (red, black) ornamental
-                        band; a cross.</decoNote>
+                        <locus target="#113r"/>
+                        <desc>Simple coloured (red, black) ornamental
+                           band with a small cross in the middle.</desc>
+                       </decoNote>
                   </decoDesc>
+                  
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#116va">f. 116va</locus>
-                           <desc>: Donation note.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#116va">116va</locus>: &lt;...&gt;፡ መጽሐፍ፡ ዘወሀበቶ፡ ለአቡነ፡
-                              አረጋዊ፡ ወለተ፡ ሥላሴ፡ ስመ፡ አቡሃ፡ ገብረ፡ ክርስቶስ፡ ስመ፡ እኁኀ፡ ገብረ፡ ሥላሴ፡ ስመ፡ እማ፡ ወለተ፡
+                           <locus target="#116va"/>
+                           <desc type="DonationNote"/>
+                           <q xml:lang="gez">
+                              መጽሐፍ፡ ዘወሀበቶ፡ ለ<placeName ref="INS0105DD"/> አቡነ፡
+                              አረጋዊ፡ <persName ref="PRS14596WalattaSellase"/> ወለተ፡ ሥላሴ፡ ስመ፡ አቡሃ፡ ገብረ፡ ክርስቶስ፡ ስመ፡ እኁኀ፡ ገብረ፡ ሥላሴ፡ ስመ፡ እማ፡ ወለተ፡
                               ማርያም፡ ስመ፡ እኅታ፡ ወለተ፡ መድኅን፡ ወለተ፡ ሚካኤል፡ ወለተ፡ ክርስቶስ፡ ከመ፡ ይኵኖሙ፡ ለኵሎሙ፡
-                              አዝማዲሃ፡ መርሃ፡ በመንግሥተ፡ ሰማያት፡ ለዝ፡ መጽሐፍ፡ ዘሠረቆ…</quote>
-                           <quote xml:lang="en"/>
+                              አዝማዲሃ፡ መርሃ፡ በመንግሥተ፡ ሰማያት፡ ለዝ፡ መጽሐፍ፡ ዘሠረቆ፡ <gap reason="ellipsis" resp="MK"/>
+                           </q>
+                           
                            <note> The note is written in the main hand but with a thinner pen nib.
-                              It states that Walatta Śǝllāse donated the Ms. to the monastery of
-                              ʾAbuna ʾAragāwi. Subsequently, her family members are mentioned: her
+                              It states that <persName ref="PRS14596WalattaSellase">Walatta Śǝllāse</persName> donated the manuscript to the monastery of
```

Comment:
at the same time, if "Walatta Śǝllāse" is the main transcription in PRS14596WalattaSellase here you can encode  `<persName ref="PRS14596WalattaSellase"/>`, without text, and `Walatta Śǝllāse` will be printed

Reply:
Ok, I understand now the difference. Thanks!